roles/slurm-management/tasks/main.yml

```diff
@@ -195,9 +218,11 @@
   tags:
     - create_database
   register: command_result
-  retries: 2
-  delay: 1
-  until: command_result == ''
+  retries: 3
+  delay: 5
+  until:
+    - command_result.stderr == ''
+    - command_result.stdout != ''
```

Comment:
Is it possible to check for the presence of the text "Nothing new added"?

Reply:
Unfortunately, that is not sufficient. For the associations that need to get configured the script will result in
```
 Modified account associations...
```
both when something really changed as well as when nothing has changed. In addition the ```Nothing new added``` will result in an exit state 1, which is also produced when an error occurred. So the best we can do for now is to use the checks as described in this pull request. When there is a serious problem, the code produces content on STDERR. When new stuff was created or stuff got updated, there should be content on STDOUT. When there is no content on STDOUT the code crashed before it could produce content on STDERR.